docs/proposals/ktl.md

```diff
@@ -0,0 +1,243 @@
+# ktl (kubectl2)
+
+## Metadata
+
+## Table of Contents
+
+- [Title](#title)
+   - [Metadata](#metadata)
+   - [Table of Contents](#table-of-contents)
+   - [Summary](#summary)
+   - [Motivation](#motivation)
+   - [Reference-level explanation](#reference-level-explanation)
+   - [Graduation Criteria](#graduation-criteria)
+   - [Alternatives](#alternatives-optional)
+   - [Unresolved Questions](#unresolved-questions-optional)
+
+## Summary
+
+- Stop trying to wholesale move kubectl out of kubernetes/kubernetes
+  - we have been working on this for a year with limited success
+- Instead, create a new cli command called `ktl`.
```

Comment:
Can you do a pitch here about why ktl or whatever we call it is better?  This is mostly a "we have to refactor stuff because refactoring is good" proposal - adding a vision statement is really important to justify the cost we'll be taking on and adding to users.

> In order to evolve the command line experience for Kubernetes, and benefit from lessons learned about how to best support declarative configuration of applications AND easy extension of infrastructure primitives, we propose a new CLI tool that will be simpler, easier to work with and extend, drive improvements to core user actions, and still preserve existing user support via kubectl.

Reply:
also justify the possible cost for user to switch to this new command line tool.